### PR TITLE
ref(discover): Extract getQueryByType function

### PR DESCRIPTION
**Pull request by lynnagara**
This will make it easier to add additional query types in the future

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx [hunk 1]
```diff
@@ -193,6 +195,38 @@ export default function createQueryBuilder(initial = {}, organization) {
...
 
+  /**
+   * Get the actual query to be run for each visualization type
+   *
+   * @param {Object} originalQuery Original query input by user (external query representation)
+   * @param {String} Type to fetch - currently either byDay or base
+   * @returns {Object} Modified query to be run for that type
+   */
+  function getQueryByType(originalQuery, type) {
+    if (type === 'byDayQuery') {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx]: Do we want these to be defined as constants somewhere?

File: src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx [hunk 2]
```diff
@@ -193,6 +195,38 @@ export default function createQueryBuilder(initial = {}, organization) {
...
+      return {
+        ...originalQuery,
+        groupby: ['time'],
+        rollup: 60 * 60 * 24,
+        orderby: 'time',
+        limit: 1000,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx]: What does this limit refer to  in this query?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx]: It's the largest number of results we can request from Snuba. Essentially if the user enters some lower number, we want to ignore that.

File: src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx]: I'll leave it for now, but might be good to follow up later when we end up with a number of these

**Pull request merged**